src/mainwindow/infomarkseditdlg.h

```diff
@@ -44,39 +44,41 @@ class InfoMarksEditDlg : public QDialog, private Ui::InfoMarksEditDlg
 public slots:
 	void objectListCurrentIndexChanged(const QString&);
 	void objectTypeCurrentIndexChanged(const QString&);
-	 
+  void objectClassCurrentIndexChanged(const QString&);
```

Comment:
I fixed the file (and configured my code editor so I don't add tabs anymore, but 4 spaces instead).

I will need to rebase this PR as it depends on PR #24. I'll wait for it to be merged, it will be easier for the review.